var/spack/repos/builtin/packages/faodel/package.py

```diff
@@ -33,6 +34,8 @@ class Faodel(CMakePackage):
     variant('network', default='nnti', values=('nnti', 'libfabric'),
             description='RDMA Network library to use for \
                          low-level communication')
+    variant('cereal', default=False,
+            description='Use Cereal to serialize NNTI data structures else XDR')
```

Comment:
```
var/spack/repos/builtin/packages/faodel/package.py:38: [E501] line too long (80 > 79 characters)
```
I know it seems counterintuitive, but if you put the entire variant on a single line, Spack will ignore it in the flake8 line length checks.

Reply:
@adamjstewart Thanks for the Pro Tip.  Is it preferred to put the entire variant on one line?  I looked at other packages and it looks like it goes both ways.

Reply:
No preference. I usually put everything on one line, other contributors usually wrap things. As long as flake8 is happy, no one will complain.

Reply:
OK.  I just pushed a new commit with each variant on a single line.